frontend/test/parsing/testParseAttributes.cpp

```diff
@@ -1054,36 +1054,33 @@ static void test18(Parser* parser) {
   assert(guard.realizeErrors() == 3);
 }
 
-// test for supporting attributes with arguments outside of parentheses
-// this is just while we are deprecating the old syntax so that
-// @unstable "message" doesn't stop working suddenly - although technically it
-// will allow ANY attribute to be written without parentheses
+// test that we reject attributes with arguments not inside parens
 static void test19(Parser* parser) {
   auto ctx = parser->context();
   ErrorGuard guard(ctx);
   std::string program = R""""(
-    @unstable "this thing is unstable"
+    @unstable category="experimental", reason="Enum is unstable", issue="82566"
     proc Foo(bar) {  }
-    @deprecated "this thing is deprecated"
+    @deprecated since="1.32", note="this thing is deprecated", suggestion="use Baz instead"
     var x: int;
-    @stable "1.28"
+    @stable since="1.28"
```

Comment:
It seems like it would be valuable to test both calls that look like this and calls that look like it did before.

Reply:
I added this and a few other test cases for attributes without parens but with single or multiple named or unnamed arguments

Reply:
Looks great, thanks!